discover/portal/articles/100-web-experience-management/02-building-a-site/03-creating-multiple-navigation-menus/01-default-navigation.markdown

```diff
@@ -0,0 +1,98 @@
+# Default Navigation
+
+The most straighforward way to manage your site navigation is to use the default
+navigation. With the default navigation, you create public and private pages, and organize those pages in whatever order or structure that you see fit.
+
+## Creating a Page
```

Comment:
The rest of the content is fine. However, there wasn't any previous content on creating and managing pages?

Reply:
We have content specifically on creating Widget Pages, Content Pages, and covering other page types, but those don't include anything regarding organization or hierarchy and I'm doing a sweep of the Documentation related to page management to make sure that any screenshots or information is updated in other docs.

Reply:
Got it. And does it make sense to have this information separate from the existing information on creating pages?

Reply:
Organizationally, because of the changes to Navigation with 7.1 (along with the ideas behind Content Pages), we had coupled the creation of pages with the creation of content in the docs, but had separated the discussion of Navigation Menus. It seemed to make sense to cover these changes alongside of Navigation Menus, but we could revisit the broader organization in the 7.2 outline.

Reply:
I see.

We need to separate content management from site building. Specially as we start promoting headless more, since for headless scenarios only the content management matters, but the site building is not relevant.

The sooner we move in this direction the better.